### ISSUE TITLE
"Disabling cuDNN frontend for the following convolution" and warning when update jax

### ISSUE THREAD

**Issue by wztdream** [google/jax#9460]
Hi,
Below simple Conv:
```
import flax.linen as nn
import jax
import jax.numpy as jnp


class Foo(nn.Module):
    @nn.compact
    def __call__(self, x):
        return nn.Conv(3, kernel_size=(3, 3), strides=(1, 1), padding="SAME")(x)


foo = Foo()
rng = jax.random.PRNGKey(0)
params_rng, input_rng = jax.random.split(rng, 2)
# x = jax.random.normal(input_rng, shape=(3, 4, 4, 3))
x = jnp.zeros(shape=(3, 4, 4, 3))
variables = foo.init({"params": params_rng}, x)


@jax.jit
def func(varaibles, x):
    return foo.apply(varaibles, x)


out = func(variables, x)
print(jax.tree_map(lambda x: x.shape, out))
```
 will trigger `Disabling cuDNN frontend for the following convolution` ... `because it uses an identity activation`, similar with #9455.
```
2022-02-06 21:04:19.614712: E external/org_tensorflow/tensorflow/stream_executor/cuda/cuda_dnn.cc:5205] Disabling cuDNN frontend for the following convolution:
  input: {count: 3 feature_map_count: 3 spatial: 4 4  value_min: 0.000000 value_max: 0.000000 layout: BatchDepthYX}
  filter: {output_feature_map_count: 3 input_feature_map_count: 3 layout: OutputInputYX shape: 3 3 }
  {zero_padding: 1 1  pad_alignment: default filter_strides: 1 1  dilation_rates: 1 1 }
  ... because it uses an identity activation.
(3, 4, 4, 3)
```
Through the output is still correct but I suppose there is something wrong. I tried to upgrade jax to latest version following the installation guide ,
```
pip install --upgrade pip

# Installs the wheel compatible with Cuda >= 11.1 and cudnn >= 8.2
pip install jax[cuda11_cudnn82] -f https://storage.googleapis.com/jax-releases/jax_releases.html
```
but there is a warning:
```
pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html
Looking in indexes: https://mirrors.bfsu.edu.cn/pypi/web/simple
Looking in links: https://storage.googleapis.com/jax-releases/jax_releases.html
Requirement already satisfied: jax[cuda] in /home/wangzongtao/anaconda3/envs/flax/lib/python3.9/site-packages (0.2.28)
warning: missing-index-doctype

× The package index page being used does not have a proper HTML doctype declaration.
╰─> Problematic URL: https://storage.googleapis.com/jax-releases/jax_releases.html

note: This is an issue with the page at the URL mentioned above.
hint: You might need to reach out to the owner of that package index, to get this fixed. See https://github.com/pypa/pip/issues/10825 for context.
```
So, how to solve this error/warning? Or is the installation guide need to update?
relation package version on by side:
```
>> nvcc --version                                                                                                                                                                                                                                                 
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Sun_Aug_15_21:14:11_PDT_2021
Cuda compilation tools, release 11.4, V11.4.120
Build cuda_11.4.r11.4/compiler.30300941_0
```
```
libcudnn_cnn_infer.so.8 -> libcudnn_cnn_infer.so.8.2.4
```
```
Ubuntu 20.04.3 LTS x86_64
```

**Comment by wztdream** [comment]
I can confirm that:
1. Convolution with relu activation will not trigger this error, on the other hand no activation, sigmoid actvation, softmax actitvation WILL trigger the error
2. Down grade jax verion to 0.2.27 and jaxlib to 0.1.75, no such error
3. I also upgrade cudnn to 8.3.x but no help

Does anyone know what this warnning mean? Is it safe to ignore them? Why indentity/sigmoid/softmax vs relu activation matter here?

**Comment by wztdream** [comment]
@hawkinsp
Is this related with new features of jaxlib0.1.76?

> New features

  >  Includes precompiled SASS for NVidia compute capability 8.0 GPUS
    (e.g. A100). Removes precompiled SASS for compute capability 6.1 so as not
    to increase the number of compute capabilities: GPUs with compute capability
    6.1 can use the 6.0 SASS.
    With jaxlib 0.1.76, JAX uses the MHLO MLIR dialect as its primary target compiler IR
    by default.

**Comment by hawkinsp** [comment]
I think it's a benign log message. It simply means a suboptimal code path was taken. It is safe to ignore it; I'll send a change asking to demote it to something that doesn't get printed.

**Comment by apaszke** [comment]
This should be fixed now (thanks to the linked commit in TF repo)!
